public/video-ui/src/components/ScheduledLaunch/ScheduledLaunch.js

```diff
@@ -250,26 +251,45 @@ export default class ScheduledLaunch extends React.Component {
     </button>
   )
 
-  renderSchedulerButton = (showScheduleOptions) => (
-    <button
-      className="btn btn--list"
-      onClick={() =>
-        this.setState({
-          showScheduleOptions: !showScheduleOptions,
-          propertyName: null
-        })
+  renderSchedulerButton = (showScheduleOptions) => {
+
+    const notAbleToScheduleReason = (function () {
+      if (!this.props.video.title) {
+        return 'You must add a title before scheduling';
       }
-    >
-      <Icon icon="access_time" />
-    </button>
-  )
+
+      if (this.props.hasPublishedVideoPageUsages()) {
+        return 'The atom cannot be scheduled because it has a published video page';
+      }
+
+      return null;
+
+    }.bind(this)());
```

Comment:
Why do we need to `.bind`? 😱

Reply:
Because I couldn't figure out the syntax to call an arrow function as I was defining it and then I decided to use bind to have access to `this.props`...

Reply:
we could move this function to a higher scope on the component itself?

```js
export default class ScheduledLaunch extends React.Component {
  getNoScheduleReason() {
    if (!this.props.video.title) {
      return 'You must add a title before scheduling';
    }
    ...
  }
  renderSchedulerButton(showScheduleOptions) {
    const notAbleToScheduleReason = this.getNoScheduleReason();
    const disabled = !!notAbleToScheduleReason;
    return (
      <div data-tip={notAbleToScheduleReason}>
        ...
      </div>
    );
  }
}
```